torch/_inductor/utils.py

```diff
@@ -31,6 +31,9 @@
 
 VarRanges = Dict[sympy.Expr, sympy.Expr]
 
+extension_scheduling: Dict[str, type] = {}
```

Comment:
utils.py seems like the wrong place for this registration stuff.  Either a new file or scheduler.py.